src/cases/components/role-access-section/role-access-section.component.html

```diff
@@ -0,0 +1,26 @@
+<h2 class="govuk-heading-m">{{title}}</h2>
+<div *ngIf="!namedRoles">
+  <exui-case-roles-table [roles]="roles"
+                        [roleCategory]="roleCategory"
+                        [caseDetails]="caseDetails"
+                        [showAllocateRoleLink]="showAllocateRoleLink">
+  </exui-case-roles-table>
+</div>
+<div *ngIf="namedRoles">
+  <exui-case-roles-table [roles]="namedRoles"
```

Comment:
@connorpgpmcelroy code could be refactored here to have only one <exui-case-roles-table> e.g:

`<exui-case-roles-table [roles]="namedRoles ? namedRoles : roles">...`

Reply:
I'll investigate this - previously had problems with template not registering changes before but if suggested changes work I'll replace

Reply:
Have replaced this, although not directly on this component but the parent component and seems to be working